File: try/spark-java-try/src/main/java/InitializingSpark.java
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;

import java.util.Arrays;
import java.util.List;

public class InitializingSpark {
    public static void main(String[] args) {
        String appName = "";
        String master = "";
        SparkConf conf = new SparkConf().setAppName(appName).setMaster(master);
        JavaSparkContext sc = new JavaSparkContext(conf);
        //Parallelized Collections
        List<Integer> data = Arrays.asList(1, 2, 3, 4, 5);
        JavaRDD<Integer> distData = sc.parallelize(data);

    }

}

File: try/scala_try/src/main/scala/course/InitializingSpark.scala
package course

import org.apache.spark.{SparkConf, SparkContext}

object InitializingSpark {

  def main(args: Array[String]): Unit = {
    val appName = " "
    val master = ""
    val conf = new SparkConf().setAppName(appName).setMaster(master)
    val sc = new SparkContext(conf)
    //Parallelized Collections
    val data = Array(1, 2, 3, 4, 5)
    val distData = sc.parallelize(data)

  }


}
